### PR TITLE
Drop arf_gibills, csv_files, csv_storages, data_csvs, va_crosswalks

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Add migrations to remove old tables if they exist

## Testing done
Successful run of `bundle exec rake db:migrate`

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

File: db/schema.rb [hunk 1]
```diff
@@ -494,23 +370,23 @@
...
-    t.index ["facility_code"], name: "institutions_temp_facility_code_idx"
-    t.index ["institution"], name: "institutions_temp_institution_idx"
-    t.index ["institution_type_name"], name: "institutions_temp_institution_type_name_idx"
-    t.index ["online_only"], name: "institutions_temp_online_only_idx"
-    t.index ["ope"], name: "institutions_temp_ope_idx"
-    t.index ["ope6"], name: "institutions_temp_ope6_idx"
-    t.index ["state"], name: "institutions_temp_state_idx"
-    t.index ["stem_offered"], name: "institutions_temp_stem_offered_idx"
-    t.index ["version", "parent_facility_code_id"], name: "institutions_temp_version_parent_facility_code_id_idx"
-    t.index ["version"], name: "institutions_temp_version_idx"
```
Comment (jholton) [inline db/schema.rb]: Remove these index renamings please.  We'll handle those in a separate PR.

File: db/schema.rb [hunk 2]
```diff
@@ -634,23 +510,22 @@
...
-    t.index ["facility_code"], name: "index_institutions_on_facility_code"
-    t.index ["institution"], name: "index_institutions_on_institution"
-    t.index ["institution_type_name"], name: "index_institutions_on_institution_type_name"
-    t.index ["online_only"], name: "index_institutions_on_online_only"
-    t.index ["ope"], name: "index_institutions_on_ope"
-    t.index ["ope6"], name: "index_institutions_on_ope6"
-    t.index ["state"], name: "index_institutions_on_state"
-    t.index ["stem_offered"], name: "index_institutions_on_stem_offered"
-    t.index ["version", "parent_facility_code_id"], name: "index_institutions_on_version_and_parent_facility_code_id"
-    t.index ["version"], name: "index_institutions_on_version"
```
Comment (jholton) [inline db/schema.rb]: Remove these index renamings please.  We'll handle those in a separate PR.

File: db/schema.rb [hunk 3]
```diff
@@ -1687,7 +1547,8 @@
     t.boolean "stem_indicator", default: false
     t.string "campus_type"
     t.string "parent_facility_code_id"
-    t.index ["facility_code"], name: "index_weams_on_facility_code", unique: true
+    t.integer "csv_row"
+    t.index ["facility_code"], name: "index_weams_on_facility_code"
```
Comment (jholton) [inline db/schema.rb]: Looks like this sneaked in from the other PR.  Remove please.

File: db/schema.rb (hunk 1 above)
Comment (zurbergram) [reply db/schema.rb]: This happens when running `bundle exec rake db:migrate`

File: db/schema.rb (hunk 3 above)
Comment (zurbergram) [reply db/schema.rb]: removed

File: db/schema.rb (hunk 2 above)
Comment (zurbergram) [reply db/schema.rb]: removed

File: db/schema.rb (hunk 1 above)
Comment (zurbergram) [reply db/schema.rb]: removed

**Comment by jholton** [comment]
This PR drops tables that shouldn't exist.  They currently do in staging and production.
The tables were created by migrations early in development.  Then the migrations were deleted from the code base [here](https://github.com/department-of-veterans-affairs/gibct-data-service/commit/4c0da20d61ea5e69fac87e22012c3230f0a8142a) and [here](https://github.com/department-of-veterans-affairs/gibct-data-service/commit/b5247e52c8517dbd3e96a83eef27fba27d3ae454) instead of being dropped via migration.

These migrations are intended to be "one way".  A rollback path that restores them is not desired.

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**